index.html

```diff
@@ -0,0 +1,323 @@
+        <li class="hcj">
+          <p>html</p>
+        </li>
+        <li class="hcj">
+          <p>Ruby on rails</p>
+        </li>
+        <li class="hcj">
+          <p>css</p>
+        </li>
+        <li class="hcj">
+          <p>javascript</p>
+        </li>
+      </ul>
+      <div class="sees"><button class="buttons" type="button">
+          <p>See Project</p>
+        </button></div>
+      </div>
+    </section>
+
+    <section class="middle-section">
+      <div class="sidew">
+      <h2 class="header" id="title">Uber Navigation</h2>
+      <ul class="details">
+        <li>
+          <div class="canopy">
+            <p>Uber</p>
+          </div>
+        </li>
+        <li>
+          <div class="counter"><img alt="#" src="./Icons/Counter.png"></div>
+        </li>
+        <li>
+          <div class="spec">
+            <p>Lead Developer</p>
+          </div>
+        </li>
+        <li>
+          <div class="counter"><img alt="#" src="./Icons/Counter.png"></div>
+        </li>
+        <li>
+          <div class="spec">
+            <p>2018</p>
+          </div>
+        </li>
+      </ul>
+      <div class="infom">
+        <p>A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
+      </div>
+      <ul class="lang">
+        <li class="hcj">
+          <p>html</p>
+        </li>
+        <li class="hcj">
+          <p>Ruby on rails</p>
+        </li>
+        <li class="hcj">
+          <p>css</p>
+        </li>
+        <li class="hcj">
+          <p>javascript</p>
+        </li>
+      </ul>
+      <div class="see"><button class="button" type="button">
+          <p>See Project</p>
+        </button></div>
+      </div>
+      <div class="sidep">
+        <img class="tonic" alt="#" src="./Work/4.png">
+        </div>
+    </section>
+
+    <section class="about-sec">
+      <div class="sidea">
+      <div >
+        <h3 class="abt">About Myself</h3>
+        <p class="info-abt">Hello I’m a software developer! I can help you build a product , feature or website Look
+          through some of my work and experience! If you like what you see and have a project you need coded, don’t
+          hestiate to contact me.</p>
+        <p class="connect">LET'S CONNECT</p>
+      </div>
+      <ul class="abt-icons">
+        <li class="cat"><a href="#"><img alt="#" src="./Icons/cat.png"></a></li>
+        <li class="linkdin"><a href="#"><img alt="#" src="./Icons/Linkedin icon.png"></a></li>
+        <li class="p"><a href="#"><img alt="#" src="./Icons/Peace.png"></a></li>
+        <li class="twitter"><a href="#"><img alt="#" src="./Icons/Twitter.png"></a></li>
+        <li class="m"><a href="#"><img alt="#" src="./Icons/M.png"></a></li>
+      </ul>
+      <div class="resume"><button class="buttonr" type="button">
+          <p>Get my resume</p>
+        </button></div>
+      </div>
+      <div class="sidel">
+      <ul class="options">
+        <li>
```

Comment:
- [ ] The entire project is not responsive i.e not displaying the mobile design correctly. You should use `CSS media query` to make the page responsive.  At a screen size of `375px` it still appears like a desktop. 

Your work at Mobile view - 375px
![image](https://user-images.githubusercontent.com/84156714/220791478-feab91ce-1538-489f-9931-822e2940fe91.png)
Below is how it should appear.
![image](https://user-images.githubusercontent.com/84156714/220791646-2a7e9ea2-f484-400b-b726-1c7728330433.png)